### PR TITLE
Added unit tests for ‘convert’ and ‘validate’ commands

### PR DESCRIPTION
**Pull request by IjzerenHein**
Also added
- .editorconfig which specifies that editors should use spaces for this project
- yarn.lock

What remains to be done is to add the `npm run test` command to `.travis.yml` but, I didn't want to do this yet, because it depends on the `feature/json-converter` PR.

Additionally, I added a test that checks whether the CLI fails when no command is specified. But it currently doesn't fail in this case and I feel it should.

### CODE REVIEW

**Comment by alrik** [comment]
Super cool! `feature/json-converter` is merged. I think you can now adjust the `travic.yml`.

I created an issue #9 for fixing the return code of the cli tool, when being invoked without any command.

**Comment by IjzerenHein** [comment]
Hey, I've rebased the commit and disabled the missing/invalid test for now.
I think it can be merged now.

File: test/base.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/*
+const { run, assert } = require('./util');
+
+describe('commands', () => {
+  it('should fail when invalid command is specified', () => {
```
Comment (alrik) [inline test/base.spec.js]: Lets [skip this test](https://mochajs.org/#inclusive-tests), rather then comment it out.

File: test/validate.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const { releaseNotes1 } = require('./assets');
+
+describe('validate', () => {
+  it('should succeed when no file is specified and ./release-notes.yml exists', () => {
+    return assert.isFulfilled(run('validate'));
+  });
+  it('should succeed when valid file is specified', () => {
+    return assert.isFulfilled(run(`validate ${releaseNotes1.yamlPath}`));
+  });
+  /* it('should fail when no file is specified but file is invalid', () => {
```
Comment (alrik) [inline test/validate.spec.js]: I'd prefer skipping tests, that test an unimplemented feature.

File: test/base.spec.js (hunk 1 above)
Comment (IjzerenHein) [reply test/base.spec.js]: Oeh, didn't know that was possible with mocha. Will make the changes

File: test/validate.spec.js (hunk 2 above)
Comment (IjzerenHein) [reply test/validate.spec.js]: Are you saying that specifying an explicit file is not supported yet? And this test case is in fact invalid and produced a false positive?

Comment (alrik) [reply test/validate.spec.js]: No, sorry i don't know where i was with my thoughts.

Passing an explicite file is supported. And this test or its setup is simply not fully implemented.
I guess you wanted to test if the command fails properly when the default `release-notes.yml` file in the current working directory is not valid.
I think we should either drop this commented out test case or provide the required setup for it.

Comment (IjzerenHein) [reply test/validate.spec.js]: Alright, I've removed the out-commented test

File: test/base.spec.js (hunk 1 above)
Comment (IjzerenHein) [reply test/base.spec.js]: Done

**Pull request merged**